.github/config.yml

```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 and Meshery community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration-Light.svg">
+  <img alt="Meshery Logo" src="">
```

Comment:
@anshgoyalevil thanks! 

Are we sure that an empty image tag works?

Reply:
Yes sir, I have manually verified that.

Reply:
Oh, good. Thank you much, @anshgoyalevil.